src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -645,12 +648,13 @@ public int recalculateLotMolWeights(List<Long> parentIds) {
 
 	@Override
 	@Transactional
-	public int restandardizeParentStructures(List<Long> parentIds)
+	public int restandardizeParentsOfStandardizationDryRunCompounds()
 			throws CmpdRegMolFormatException, StandardizerException, IOException {
 
 		int batchSize = propertiesUtilService.getStandardizationBatchSize();
 
-		int totalCount = parentIds.size();
+		int totalCount = StandardizationDryRunCompound.getReadyStandardizationChangesCount();
+		int remainingCount = totalCount;
```

Comment:
Instead of getting a giant list of parentIds (which can be a lot), we will loop until we have no more standardization dry run compounds in the READY state.